ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml

```diff
@@ -19,8 +19,15 @@ chain:
     - name: EXTRA_ARGS
       default: ""
       documentation: "Extra args to pass to the create cluster aws command"
+    - name: HYPERSHIFT_GUEST_INFRA_OCP_ACCOUNT
+      default: "false"
```

Comment:
fwiw I consider hypershift-aws-create-chain.yaml deprecated. There's no semantic difference with `hypershift-aws-setup-nested-management-cluster-chain.yaml` which we should rename to just `hypershift-aws-create. Then is up to the workload to chain multiples creates to have nested clusters.

However I'm fine with the change for now and I'll follow up later on with the refactor above.